setup.py

```diff
@@ -27,4 +27,9 @@
     packages=find_packages(exclude=["contrib", "docs", "tests"]),
     install_requires=install_reqs,
     package_dir={"shimmingtoolbox": "shimmingtoolbox"},
+    entry_points={
+        'console_scripts': [
+            'st_script_nifti = examples.script_nifti:main',
```

Comment:
These are absolute paths. As written, `example` is a new top-level python package, and will collide with anyone else who also made a new top-level python package named `example`.

In https://github.com/shimming-toolbox/shimming-toolbox-py/pull/6 you can see my example: I put the scripts under `shimmingtoolbox.cli`.